Gruntfile.js

```diff
@@ -40,10 +40,7 @@ module.exports = function (grunt) {
     /**
      * Compile tasks
      */
-    grunt.registerTask('compile', function () {
-        grunt.task.run(['clean', 'shell:collections', 'shell:config', 'replace', 'cacheBust']);
-    });
     grunt.registerTask('bundle', function () {
```

Comment:
think we're meant to keep the `compile` task rather than `bundle` - tasks look correct, just the name